usaspending_api/broker/helpers/last_delta_table_load_version.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from usaspending_api.broker import lookups
+from usaspending_api.broker.models import DeltaTableLoadVersion
+
+logger = logging.getLogger("script")
+
+
+def get_last_delta_table_load_versions(key: str):
+    """
+    Returns the Delta Table version IDs of the table, identified by the provided key, were loaded to staging
+    tables and live Postgres tables.
+
+    Args:
+        key: Lookup key that identifies the delta table
+
+    Returns:
```

Comment:
Don't we indicate the data type returned by methods as `-> <type>` . In this method's case it's returning a tuple of two ints.

Reply:
Yes, can do